proof-general-pkg.el

```diff
@@ -0,0 +1,2 @@
+(define-package "proof-general" "4.4.1~pre" "A generic front-end for proof assistants (interactive theorem provers)"
```

Comment:
@psteckler 
I think the version string in this file (that comes from Clément's PR #157) can also be changed to 5.0-git in your fork